web/src/components/modals/standardize-metadata.tsx

```diff
@@ -0,0 +1,424 @@
+                                      td.style.color = 'red';
+                                    }
+                                  }
+                                },
+                              },
+                            ]}
+                            licenseKey="non-commercial-and-evaluation"
+                            className="custom-handsontable"
+                          />
+                        </div>
+                      </div>
+                      <div className="col-6" role="group" aria-label="radio_group">
+                        <div className="w-100 h-100 rounded-2 outer-container">
+                          <div className="btn-group-vertical w-100 h-100 bg-white rounded-2">
+                            <input
+                              className="btn-check"
+                              type="radio"
+                              name={key}
+                              id={`${key}-original`}
+                              value={key}
+                              checked={selectedValues[key] === key} // Check if the selected value is the same as the key
+                              onChange={() => handleRadioChange(key, null)}
+                            />
+                            <label
+                              className="btn btn-outline-secondary selected-outline shadow-sm bg-white"
+                              htmlFor={`${key}-original`}
+                            >
+                              <strong className="fw-semibold">{key}</strong> (original value)
+                            </label>
+
+                            {Object.entries(standardizedData[key]).map(([subKey, value], index, array) => (
+                              <React.Fragment key={subKey}>
+                                <input
+                                  className="btn-check"
+                                  type="radio"
+                                  name={key}
+                                  id={`${key}-suggested-${subKey}`}
+                                  value={subKey}
+                                  checked={selectedValues[key] === subKey}
+                                  disabled={standardizedData[key]['Not Predictable'] === 0}
+                                  onChange={() => handleRadioChange(key, subKey)}
+                                />
+                                <label
+                                  className="btn btn-outline-secondary selected-outline shadow-sm bg-white"
+                                  htmlFor={`${key}-suggested-${subKey}`}
+                                >
+                                  {subKey} ({formatToPercentage(value)})
+                                </label>
+                              </React.Fragment>
+                            ))}
+                          </div>
+                        </div>
+                      </div>
+                      <br />
+                    </div>
+                  </div>
```

Comment:
This page is already pretty huge... Could we pull this out and make it a separate component?

Reply:
fixed